src/common/constants/dev-config.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "dev",
   "announcement": {
-    "message": "On August 21st, 2020, the Bureau released the 2021 FIG and the Supplemental Guide for Quarterly Filers for 2021.",
+    "message": "The filing period is open. Submissions of 2020 HMDA data will be considered timely if received on or before Monday, March 1st, 2021.",
```

Comment:
@meissadia Lets get commits specific to this file merged in (maybe a separate PR) so that we can start accessing the 2020 platform in our dev